### PR TITLE
Additional params for publish layer version api

### LINKED ISSUE #4: Feature: Accept additional parameters for PublishLayerVersion API
Examples: license, description, etc

### PR DESCRIPTION
**Pull request by asahasrabuddhe**
Resolves #4 

Added support for description and license when publishing layers

Before:

screenshot from 2019-01-25 17-04-32
screenshot from 2019-01-25 17-04-34

After:

screenshot from 2019-01-25 17-04-35
screenshot from 2019-01-25 17-04-36
screenshot from 2019-01-25 17-08-23

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,11 +29,14 @@ USAGE:
    img2lambda [options]
 
 GLOBAL OPTIONS:
-   --image value, -i value             Name of the source container image. For example, 'my-docker-image:latest'
+   --image value, -i value             Name of the source container image. For example, 'my-docker-image:latest'. The Docker daemon must be pulled locally already.
```
Comment (clareliguori) [inline README.md]: Correction: The Docker *image* must be pulled locally already.

File: README.md [hunk 2]
```diff
@@ -29,11 +29,14 @@ USAGE:
...
    --region value, -r value            AWS region (default: "us-east-1")
    --output-directory value, -o value  Destination directory for command output (default: "./output")
    --layer-namespace value, -n value   Prefix for the layers published to Lambda (default: "img2lambda")
    --dry-run, -d                       Conduct a dry-run: Repackage the image, but only write the Lambda layers to local disk (do not publish to Lambda)
+   --description value, --desc value   The description of this layer version
+   --license-info value, -l value      The layer's software license. It can be an SPDX license identifier, the URL of the license hosted on the internet or the full text of the license
```
Comment (clareliguori) [inline README.md]: I am an Oxford comma person, please add a comma before 'or'

File: README.md [hunk 3]
```diff
@@ -117,9 +120,9 @@ docker build -t lambda-php .
 
 The example PHP functions are also built into the example image, so they can be run with Docker:
 ```
-docker run lambda-php hello '{"name": "World"}'
+docker run lambda-php functions.hello '{"name": "World"}'
```
Comment (clareliguori) [inline README.md]: Looks like changes from your other PR are getting into this, can you separate into different branches?

File: README.md (hunk 1 above)
Comment (asahasrabuddhe) [reply README.md]: I had copied this from the output of the actual binary. This mistake was also present in main.go which I have now fixed.

File: README.md (hunk 2 above)
Comment (asahasrabuddhe) [reply README.md]: Fixed :)

File: README.md (hunk 3 above)
Comment (asahasrabuddhe) [reply README.md]: Resolved

**Comment by asahasrabuddhe** [comment]
Hi @clareliguori !

Thanks a lot for taking time to review my code. I have fixed the concerns which you had raised above.

Regards,
Ajitem

**Pull request merged**